controllers/integration_test.go

```diff
@@ -522,9 +522,7 @@ var _ = Describe("Integration Test", func() {
 				By("Checking DW Directive Breakdown")
```

Comment:
Alignment seems off here too.

Reply:
I'm not seeing this alignment issue either.